keyboards/lily58/readme.md

```diff
@@ -0,0 +1,16 @@
+# Lily58
+
+Lily58 is 6×4+5keys column-staggered split keyboard.
```

Comment:
Putting a blank line between this line and the next will make your readme file easier to read.